test/sample.tf

```diff
@@ -10,3 +10,19 @@ resource "datadog_metric_metadata" "request_time" {
   type        = "gauge"
   unit        = "millisecond"
 }
+
+resource "postgresql_function" "orb_ci_test_function" {
+  provider = postgresql.main
+  name     = "orb_ci_test_function"
+  arg {
+    name = "i"
+    type = "integer"
+  }
+  returns  = "integer"
+  language = "plpgsql"
+  body     = <<-EOF
+        BEGIN
+            RETURN i + 1;
+        END;
+    EOF
+}
```

Comment:
Tohle je funkční test. V main db na devu se vytvoří jedna bezprizorní funkce 🙂 Funguje to:
![image](https://github.com/FigurePOS/circle-ci-node-ecs-orb/assets/215660/41bf0945-e990-401d-acca-5e4bc7cf6cbd)
(https://app.circleci.com/pipelines/github/FigurePOS/circle-ci-node-ecs-orb/264/workflows/e2294a6f-d2c4-4860-b0b8-a64599ee86a8/jobs/1831)